integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/AdministrativeDecoupleMBusDevice.feature

```diff
@@ -0,0 +1,90 @@
+    When the Administrative Decouple G-meter "TESTG102400000001" request is received
+    Then the Administrative Decouple response is "OK"
+    And the mbus device "TESTG102400000001" is not coupled to the device "TEST1024000000001"
+    And the channel of device "TESTG102400000001" is cleared
+    And the mbus primary address of device "TESTG101205673117" is cleared
+
+
+  Scenario: Administrative Decouple coupled G-meter "TESTG101205673117" from E-meter "TEST1024000000001"
+    Given a dlms device
+      | DeviceIdentification | TEST1024000000001 |
+      | DeviceType           | SMART_METER_E     |
+    And a dlms device
+      | DeviceIdentification           | TESTG101205673117 |
+      | DeviceType                     | SMART_METER_G     |
+      | GatewayDeviceIdentification    | TEST1024000000001 |
+      | Channel                        |                 1 |
+      | MbusIdentificationNumber       |          12056731 |
+      | MbusPrimaryAddress             |                 9 |
+      | MbusManufacturerIdentification | LGB               |
+      | MbusVersion                    |                66 |
+      | MbusDeviceTypeIdentification   |                 3 |
+    And device simulation of "TEST1024000000001" with M-Bus client version 0 values for channel 1
+      | MbusPrimaryAddress             | 3        |
+      | MbusIdentificationNumber       | 12056731 |
+      | MbusManufacturerIdentification | LGB      |
+      | MbusVersion                    | 66       |
+      | MbusDeviceTypeIdentification   | 3        |
+    When the Administrative Decouple G-meter "TESTG101205673117" request is received
+    Then the Administrative Decouple response is "OK"
+    And the mbus device "TESTG101205673117" is not coupled to the device "TEST1024000000001"
+    And the channel of device "TESTG101205673117" is cleared
+    And the mbus primary address of device "TESTG101205673117" is cleared
+
+  Scenario: Administrative Decouple decoupled G-meter "TESTG101205673117" from E-meter "TEST1024000000001"
+    Given a dlms device
+      | DeviceIdentification | TEST1024000000001 |
+      | DeviceType           | SMART_METER_E     |
+    And a dlms device
+      | DeviceIdentification           | TESTG101205673117 |
+      | DeviceType                     | SMART_METER_G     |
+      | MbusIdentificationNumber       |          12056731 |
+      | MbusPrimaryAddress             |                 9 |
+      | MbusManufacturerIdentification | LGB               |
+      | MbusVersion                    |                66 |
+      | MbusDeviceTypeIdentification   |                 3 |
+    And device simulation of "TEST1024000000001" with M-Bus client version 0 values for channel 1
+      | MbusPrimaryAddress             | 0 |
+      | MbusIdentificationNumber       | 0 |
+      | MbusManufacturerIdentification | 0 |
+      | MbusVersion                    | 0 |
+      | MbusDeviceTypeIdentification   | 0 |
```

Comment:
Please add some values, to verify values are not changed or set to 0

Reply:
| MbusPrimaryAddress             | 3        |
      | MbusIdentificationNumber       | 12056731 |
      | MbusManufacturerIdentification | LGB      |
      | MbusVersion                    | 66       |
      | MbusDeviceTypeIdentification   | 3        |
      
And verify these values do no changes after response

Reply:
Check